### DISCUSSION TITLE
Change Footer

### DISCUSSION THREAD

**Discussion by homie-sh** [Floorp-Projects/Floorp#690] [Feature Request]
Hey Floorp Team,
maybe you can change the **date** of the **footer** on your Website.
luca

**Comment by creeper-0910** [comment]
What does that mean?

**Comment by snirt** [comment]
That one? ![Screenshot_2024-01-22-21-09-12-67_e4424258c8b8649f6e67d283a50a2cbc.jpg](https://github.com/Floorp-Projects/Floorp/assets/20107490/5e71b85a-e6f6-41ff-b7cd-6f18eaea0a5e)

**Reply by homie-sh** [reply]
Yes, sorry for the poor explanation.

**Comment by homie-sh** [comment]
Also on the https://docs.ablaze.one/ the footer isn't updated. Maybe u can automated that.
You can automate that with a simple js script:
```js
<p>
  &copy
  <p id='copyright'></p>Ablaze.</p>

<script>
    fetch('http://worldclockapi.com/api/json/utc/now')
      .then(data => data.json())
      .then(d => 
      document.getElementById('copyright').innerHTML = (new Date(d.currentDateTime).getFullYear())
)
</script>
 ```
And also big thanks for the nice browser I really enjoy it.
